grunt-chrome-build/grunt-chrome-build.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+module.exports = function(grunt) {
+
+  grunt.registerMultiTask('grunt-chrome-build', 'Build packaged Chrome app from sources.', function() {
+    grunt.loadNpmTasks('grunt-contrib-compress');
+    grunt.loadNpmTasks('grunt-jinja');
+    
+    grunt.log.writeln('Chrome packaged app build.');
+    
+    var options = this.options();
+    // Steps:
+    // 1. Delete existing build dir if it exists.
+    var buildDir = options.buildDir;
+    if (!buildDir) {
+      throw grunt.util.error('Missing required buildDir option.');
+    }
+    grunt.log.writeln('Deleting buildDir: ' + buildDir);
+    grunt.file.delete(buildDir);
+    
+    // 2. Create build dir.
+    grunt.log.writeln('Creating empty buildDir: ' + buildDir);
+    grunt.file.mkdir(buildDir);
+    
+    // 3. Copy sources to build dir.
+    this.files.forEach(function(f) {
+      grunt.file.copy(f.src, f.dest);
+    });
+    
+    // 4. Transform template file.
+    grunt.config.set('jinja.chrome-build', {
+      options: {
+        templateDirs: ['samples/web/content/apprtc'],
+        contextRoot: 'samples/web/content/apprtc'
+      },
+      files: [{
+        src: 'samples/web/content/apprtc/index.html',
+        dest: 'build/chrome-app/appwindow.html'
+      
+      }]
+    });
+    grunt.task.run('jinja:chrome-build');
+
+    // 5. Create zip file.
+    var zipFile = options.zipFile;
+    if (!zipFile) {
+      throw grunt.util.error('Missing required zipFile option.');
+    }
+    grunt.config.set('compress.chrome-build', {
+      options: { archive: zipFile },
+      files: [
+        {
+          expand: true, 
+          cwd: buildDir,  
+          src: ['**/*']
+        }
+      ]
+    });
+    
+    grunt.task.run('compress:chrome-build');
+    
+    // This section does not work yet.
```

Comment:
Can we do this step conditionally, i.e. if chromeBinary and keyFile are provided, create crx file; otherwise, skip it.